cmd/guacone/cmd/certifier.go

```diff
@@ -30,23 +29,23 @@ import (
 	"github.com/guacsec/guac/pkg/logging"
 	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
-func init() {
-	certifierCmd.PersistentFlags().StringVar(&flags.dbAddr, "db-addr", "neo4j://localhost:7687", "address to neo4j db")
-	certifierCmd.PersistentFlags().StringVar(&flags.creds, "creds", "", "credentials to access neo4j in 'user:pass' format")
-	certifierCmd.PersistentFlags().StringVar(&flags.realm, "realm", "neo4j", "realm to connecto graph db")
-	_ = certifierCmd.MarkPersistentFlagRequired("creds")
-}
-
 var certifierCmd = &cobra.Command{
 	Use:   "certifier",
 	Short: "certifies packages in GUAC graph",
 	Run: func(cmd *cobra.Command, args []string) {
 		ctx := logging.WithLogger(context.Background())
 		logger := logging.FromContext(ctx)
 
-		opts, err := validateCertifierFlags()
+		opts, err := validateCertifierFlags(
+			viper.GetString("gdbuser"),
+			viper.GetString("gdbpass"),
+			viper.GetString("db-addr"),
+			viper.GetString("realm"),
```

Comment:
Ah - just occurred to me - we probably need to change the docs as well, for https://github.com/guacsec/guac/edit/main/SETUP.md